dbilo-siuntos-backend/Infrastructure/Services/EmailSender.cs

```diff
@@ -0,0 +1,111 @@
+﻿using Infrastructure.Interfaces;
+using Infrastructure.Models;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Google.Apis.Auth;
+using Google.Apis.Auth.OAuth2;
+using MimeKit;
+using MimeKit.Text;
+using MailKit.Net.Smtp;
+using MailKit.Security;
+
+
+namespace Infrastructure.Services
+{
+    public class EmailSender : IEmailSender
+    {
+        private readonly ILogger<EmailSender> _logger;
+        private readonly Secrets _secrets;
+        public EmailSender(ILogger<EmailSender> logger, Secrets secrets)
+        {
+            _logger = logger;
+            _secrets = secrets;
+        }
+        public async void SendEmail(Address data, string tracking, bool isReceiver)
```

Comment:
Actually disregard the comment on the bottom - I think it would be better to have separate send email methods for every type of email

Reply:
Done